Directory.Build.props

```diff
@@ -6,7 +6,7 @@
 
   <PropertyGroup Label="Package Versions">
     <VersionPrefix>3.0.0</VersionPrefix>
-    <NpgsqlVersion>4.0.7</NpgsqlVersion>
+    <NpgsqlVersion>4.1.0-ci.1909</NpgsqlVersion>
```

Comment:
I've pinned this package on myget to prevent it from getting deleted. At some point we should consider releasing a preview1 to nuget.org.